src/ETISS.cpp

```diff
@@ -776,15 +796,195 @@ void etiss_initialize(std::vector<std::string> args, bool forced = false)
+
+            if (vm.count("help")) {
+                std::cout << desc;
+                std::cout << "\nPlease begin all options with --\n";
+            }
+
+            auto unregistered = po::collect_unrecognized(parsed_options.options, po::include_positional);
+            for (auto iter_unreg : unregistered)
+            {
+                if (iter_unreg.find("-i") != 0 )
+                {
+                    std::cout << "\nUnrecognised option "<< iter_unreg<< ".\n";
+                    std::cout << "Please use --help to list all recognised options. \n\n";
+                }
+            }
+        }
+    catch(std::exception& e) {
+        std::cout << e.what() << "\n";
+    }
+    }
+    etiss_loadIniConfigs();
+
+    // log level
+    {
+        int ll = cfg().get<int>("loglevel", etiss::WARNING);
+        if (ll >= 0 && ll <= etiss::VERBOSE)
+        { // valid log level
+          // dnm
+            // etiss::verbosity() = etiss::VERBOSE;
+            etiss::verbosity() = (Verbosity)ll;
+            etiss::log(etiss::VERBOSE, "Log level set to VERBOSE");
+        }
+        else
+        {
+            etiss::verbosity() = etiss::WARNING;
+            etiss::log(etiss::ERROR, "Specified log level is not valid. must range between 0 (= "
+                                     "silent) and 5 (= verbose)");
+        }
+    }
+
+    etiss::py::init(); // init python
+
+    // configure console
+    if (cfg().get<bool>("sigint-console", false))
+    {
+        etiss_initialize_SIGINT();
+    }
+    else
+    {
+        etiss_remove_SIGINT();
+    }
+
+    // load all found libraries
+    if (cfg().get<bool>("library-loading", true))
+    {
+        preloadLibraries();
+    }
+
+    // load integrated library
+    if (cfg().get<bool>("integrated-library", true))
+    {
+        etiss::addLibrary(LibraryInterface::openIntegratedLibrary());
+    }
+
+    // check if some required files can be found
+    {
+        std::string path = etiss::installDir();
+        std::vector<std::string> requiredFiles;
+
+        // required files
+        requiredFiles.push_back(path + "/include/jit/etiss/jit/CPU.h");
+
+        // check
+        for (auto iter = requiredFiles.begin(); iter != requiredFiles.end(); iter++)
+        {
+            std::ifstream f(iter->c_str());
+            if (!f)
+            {
+                etiss::log(etiss::WARNING, std::string("Could not find file: ") + *iter + "\n" +
+                                               "\t The installation seems broken");
+            }
+        }
+    }
+
+    // load fault files
+    {
+        std::string faults = cfg().get<std::string>("faults.xml", "");
+        if (!faults.empty())
+        {
+            std::list<std::string> ffs = etiss::split(faults, ';');
+            for (auto ff : ffs)
+            {
+                etiss::fault::Stressor::loadXML(ff);
+            }
+        }
+    }
+
+    if (cfg().get<bool>("pyconsole", false))
+        etiss::py::console();
+}
```

Comment:
Since there is a lot of duplicate code here, the original initialize function should be rmeoved.